### ISSUE TITLE
Support Jupyter slide decks

### ISSUE THREAD

**Issue by OmeGak** [indico/indico#2149]
Related ticket in SNOW `CHG34058`.

Apparently it's possible to do slides in Jupyter notebooks, here's an [example](http://nbviewer.ipython.org/format/slides/github/glouppe/tutorial-sklearn-lhcb/blob/master/An%20introduction%20to%20Machine%20Learning%20with%20Scikit-Learn-Rendered.ipynb#/2).

**Comment by pferreir** [comment]
How widely used is this? And isn't Jupyter supposed to be a REPL-based documentation tool?
I am in favour of a speakerdeck.com style solution that integrates with Indico, but this may be a bit too much for the moment.

**Comment by betatim** [comment]
It is used a lot, in particular for tutorials. People use the notebook to directly integrate the executable code examples with their slides which show information. See for example: http://nbviewer.ipython.org/format/slides/github/glouppe/tutorial-sklearn-dslhc2015/blob/master/An%20introduction%20to%20Machine%20Learning%20with%20Scikit-Learn.ipynb#/ (you can switch between slide and notebook layout)

Would it be possible to have a nbviewer instance which is protected by indico credentials? This would also solve #2148

**Comment by betatim** [comment]
Actually the slide view on nbviewer seems to be broken with this particular example. Not particularly good advertisement ...

**Comment by pferreir** [comment]
@betatim I totally agree with that (and have suggested it in the past), but that's a solution that can be provided independently from Indico and then integrated with it.

**Comment by pferreir** [comment]
Indico can work as an OAuth provider. That could be a solution.

But I guess that then you would like Indico to tell you whether the user should be able to access that particular file. I don't think our API supports that right now, but it looks like something that could be done.

**Comment by cranmer** [comment]
Hi

I don't understand the argument against having slides. Speaker-deck isn't executable. The separation of the study (the notebook) and the presentation (the paper / slides) is the origin of many reproducibility problems. So merging them is good. The fact that the notebook is rendered as slides is a presentation detail.

**Comment by pferreir** [comment]
@cranmer The problem is not so much whether it's executable or not. It's more whether that wouldn't belong in a different service, transparently linked with Indico, Zenodo, CDS, etc... I see great potential there, but I think it's time to think about the possibility of a common solution.
